src/ivc/cyclefold/sfc/input/assigned.rs

```diff
@@ -430,12 +449,55 @@ impl<F: PrimeField> SangriaAccumulatorInstance<F> {
 
 pub type SangriaCrossTermCommits<F> = Vec<(AssignedValue<F>, AssignedValue<F>)>;
 
+pub struct PairedIncoming<F: PrimeField> {
```

Comment:
NBD: add a short comment to describe what is it, e.g. something like
// instance from support circuit

Reply:
Ok, will add at the end of PR-train ✍️